packages/TelegramClient-Core.package/TCCSuperGroupChat.class/instance/loadInfo.st

```diff
@@ -1,6 +1,7 @@
 loading
 loadInfo
 
-	self
-		loadGeneralInfo;
-		loadMembers.
\ No newline at end of file
+	self loadGeneralInfo.
+	self isChannel ifFalse: [
```

Comment:
`self isNotChannel` would be nicer

Reply:
or `self canLoadMembers`

Reply:
since it is the only place I would suggest leaving it as it is

Reply:
ok, fair enough